pkg/controllers/capr/managesystemagent/managesystemagent.go

```diff
@@ -280,6 +280,12 @@ func installer(cluster *rancherv1.Cluster, secretName string) []runtime.Object {
 				},
 			},
 			ServiceAccountName: systemAgentUpgraderServiceAccountName,
+			Secrets: []upgradev1.SecretSpec{
+				{
+					Name: "stv-aggregation",
+					Path: fmt.Sprintf("/host%s/env", capr.GetSystemAgentDataDir(&cluster.Spec.RKEConfig.RKEClusterSpecCommon)),
```

Comment:
https://github.com/rancher/rancher/pull/48569/files#diff-2d6a56c7215bb70f2f4af658fd18d81ccf71dd7774f92c37d987a608703515cfR295 still exists. I can remove if necessary but I am somewhat cautious to do so until at least a minor version passes.